tests/h/presenters/annotation_searchindex_test.py

```diff
@@ -7,12 +7,9 @@
 from h.services.annotation_moderation import AnnotationModerationService
 
 
-@pytest.mark.usefixtures(
-    "DocumentSearchIndexPresenter", "moderation_service", "thread_ids"
-)
+@pytest.mark.usefixtures("nipsa_service")
 class TestAnnotationSearchIndexPresenter:
```

Comment:
This was refactored quite a bit first and then then test was added. These are in two separate commits.